src/Workspaces/CSharp/Portable/Extensions/SyntaxTriviaListExtensions.cs

```diff
@@ -47,6 +48,34 @@ public static IEnumerable<SyntaxTrivia> SkipInitialWhitespace(this SyntaxTriviaL
             return triviaList.SkipWhile(t => t.Kind() == SyntaxKind.WhitespaceTrivia);
         }
 
+        private static IEnumerable<List<SyntaxTrivia>> GetLines(this SyntaxTriviaList triviaList)
+        {
+            var currentLine = new List<SyntaxTrivia>();
```

Comment:
ArrayBuilder.GetInstance().

Reply:
What do you mean?

Reply:
Use ArrayBuilder.GetInstance to make a builder for an immutbale array.  we don't lke passing around mutable data like this.  ImmutableArrays are far preferred as they are efficient and make it plainly clear that things cannot be mutated by any other party.